CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.8.0
+<!-- generated comparing v5.7.0..master -->
+_Apr 4, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add the documentation to save and restore the grid state (#4028) @flaviendelangle
```

Comment:
```suggestion
- 🚀 Add methods to save and restore the grid state (#4028) @flaviendelangle
```

Since we did never communicate about it, I would not limit the highlight to the doc part
Or at least, make clear that the feature is new:

```suggestion
- 🚀 Publicly expose new methods to save and restore the grid state (#4028) @flaviendelangle
```